packages/react/src/components/Popup/PopupContent.tsx

```diff
@@ -53,6 +53,12 @@ export interface PopupContentProps
 
   /** A ref to a pointer element. */
   pointerRef?: React.Ref<Element>
+
+  /**
+   * Indicates that PopupContent is wrapped with FocusZone. Do not use it, it used only for internal implementation and
```

Comment:
nit: lets use @deprecated pragma to make proper accent on that - as this pragma is commonly supported and visualised (at least) by IDEs

Reply:
Will do, but you accidentally tagged someone 😸